nature/templates/nature/reports/species-report.html

```diff
@@ -28,7 +28,10 @@
     <div class="row">
         <b class="col-sm-2">{% trans "Species" %}</b>
         <div class="col-sm-10">
-            <i>{{ species.name_sci_1 | lower | capfirst }} {{ species.name_subspecies_1 | lower }}</i>
+            <i>
+                {% if species.name_sci_1 %}{{ species.name_sci_1 | lower | capfirst }}{% endif %} 
```

Comment:
Fine adjusting: Using default-tag is shorter `|default:""` than using if-statement.